ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -157,6 +152,17 @@ module.exports = Marionette.LayoutView.extend({
             }
         }
     },
+    handlePolygonDeserialization(filter) {
+        const polygonArray = (filter.value && filter.value.value && CQLUtils.arrayFromPolygonWkt(filter.value.value)) || [];
+        const bufferWidth = filter.polygonBufferWidth || filter.distance;
+
+        this.model.set({
+            mode: 'poly',
+            polygon: filter.polygon || polygonArray,
+            ...(bufferWidth && {polygonBufferWidth: bufferWidth})
```

Comment:
❓ Do we need to do the check here?

Reply:
The former, I was wondering why we needed to leave off polygonBufferWidth in some cases.

Reply:
@andrewkfiedler I did this mainly out of caution so as not to create a regression if something else expected this object as only `{mode: 'poly', polygon: ...}` without the `polygonBufferWidth`